models/gpg_key.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"time"
+
+	"github.com/gogits/gogs/modules/log"
+)
+
+//TODO maybe refector with ssh-key ?
+//TODO database behind
+
+// PublicGPGKey represents a GPG key.
+type PublicGPGKey struct {
+	ID          int64  `xorm:"pk autoincr"`
+	OwnerID     int64  `xorm:"INDEX NOT NULL"`
```

Comment:
humm? AFAIK this allows for a one-to-many setup, so a single user can have many keys.